### ISSUE TITLE
Kimogila Fighter Reference card

### ISSUE THREAD

**Issue by ImTheRhino** [tjakubo2/TTS_xwing#58]
Currently the squad builder spawns the K-Wing Ref card, not the Kimogila

**What happened:**
Incorrect Redcard
**What was supposed to happen:**
Correct Refcard
**How to make the issue happen:**
Squad builder button
**Does this still occur after saving & loading game:**
Yes

**Comment by tjakubo2** [comment]
Yeah, that also happens for other new ships. As per update notes on Discord:

- *Warning:* Refcards for new ships are duds. I'm reconsidering approach on refcards since it's always hard to find/make them.

I'll post on this issue once something advances in that matter :)

**Comment by JohnnyCheese** [comment]
Check this site, they have up to date reference cards

http://www.ftbsighters.com/xwing-maneuvers.aspx

**Comment by tjakubo2** [comment]
I'll be trying to have those implemented in a sane way since we got custom deck spawning feature in the newest TTS update. The problem is, it doesn't work for me for now.
